### ISSUE TITLE
Processing multiple images  at the same time with GPU

### ISSUE THREAD

**Issue by terryll** [AlturosDestinations/Alturos.Yolo#19]
I notices that the GPU utilization of GeForce GTX 1060 is only 12%. The frame rate is 26 per frame. Could I double the frame rate by processing two images in parallel ?
I created two instants of yolowrapper a run two tasks asynchronously. The problem is getting some GPU memory issues. Any thought and suggestion would be greatly appreciated.

**Comment by tinohager** [comment]
If you have questions about the performance please contact directly [AlexeyAB](https://github.com/AlexeyAB/darknet)

**Comment by terryll** [comment]
Thanks, I will try that. 
Is there way to label the "New issue" as question ? 
Thanks,
